app/scripts/lib/seed-phrase-verifier.js

```diff
@@ -1,4 +1,4 @@
-import KeyringController from 'eth-keyring-controller';
+import { KeyringController } from 'eth-keyring-controller';
```

Comment:
`KeyringController` is now a named export: https://github.com/MetaMask/KeyringController/blob/main/CHANGELOG.md?plain=1#L17